99-general-concepts/03-style-guide/01-style-guide/README.md

```diff
@@ -8,8 +8,111 @@
 
 ## Objetivos de Aprendizaje
 
-- ...
+- Usar una guía de estilos para escribir código más legible.
 
 ***
 
-...
+Una guía de estilo es una lista de reglas para formatear tu código. Muchas
+empresas usan una guía de estilo para que toda su base de código parezca escrita
+por el mismo desarrollador, en lugar de que los desarrolladores diseñen el
+código de acuerdo con sus propias preferencias. Esto hace que el código sea más
+fácil de leer para todos.
+
+## La guía de estilos Laboratoria
+
+En Laboratoria, hemos preparado una guía de estilos para que la sigas a partir
+de ahora en todos tus ejercicios, lo puedes encontrar [aquí](https://github.com/Laboratoria/js-style-guide). Por favor léelo ahora.
+
+Por favor, intenta aplicar todas las reglas de la guía Laboratoria en su código
```

Comment:
I think this line is redundant with the paragraph above. What do you think?

Reply:
I don't think so? The paragraph above just tells them to read the guide - this tells them to use it from now on. Or at least, that's what I meant to say.

Reply:
But it says: `para que la sigas a partir de ahora en todos tus ejercicios`